lms/templates/instructor/instructor_dashboard_2/cohort-editor.underscore

```diff
@@ -12,7 +12,7 @@
         </h3>
         <div class="cohort-management-group-setup">
             <div class="setup-value">
-                <% if (cohort.get('assignment_type') == "none") { %>
+                <% if (cohort.get('assignment_type') == "manual") { %>
```

Comment:
the `RANDOM` and `MANUAL` pseudo-constants in Python are nice, any way we can get something similar in JS ?